unable to install @expo/next-adapter

### Summary

Hi, i'm trying to install @expo/next-adapter following the instructions from the official docs.
Having expo@47.0.0.

I'm getting the error below. What can i do in this case?
Thank you.


### Environment

(details that may be relevant)
System:
      OS: macOS 12.4
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 14.19.0 - ~/n/bin/node
      npm: 6.14.16 - ~/n/bin/npm
      Watchman: 2022.11.14.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
...
    npmPackages:
      @expo/webpack-config: ^0.17.4 => 0.17.4 
      babel-preset-expo: ~9.2.1 => 9.2.2 
      expo: ^47.0.0 => 47.0.8 
      react: 18.1.0 => 18.1.0 
      react-dom: 18.1.0 => 18.1.0 
      react-native: 0.70.5 => 0.70.5 
      react-native-web: ~0.18.7 => 0.18.10 
    npmGlobalPackages:
      eas-cli: 2.7.1
      expo-cli: 6.1.0
    Expo Workflow: managed

### Please specify your device/emulator/simulator platform, model and version

web

### Error output

npm i --save-dev @expo/next-adapter
...
npm WARN @expo/next-adapter@5.0.0 requires a peer of expo@^46 but none is installed. You must install peer dependencies yourself.
npm WARN @expo/next-adapter@5.0.0 requires a peer of webpack@^4.46.0 || ^5.74.0 but none is installed. You must install peer dependencies yourself.
...
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /<my-project-path>/node_modules/@expo/next-adapter/echo "the next-expo command has been deprecated in favor of manual setup. See README for more." && exit 1
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/<my-project-path>/node_modules/@expo/next-adapter/echo "the next-expo command has been deprecated in favor of manual setup. See README for more." && exit 1'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
...

### Reproducible demo or steps to reproduce from a blank project

don't have any.
if needed i can give the package.json eventually.

--- a/packages/next-adapter/package.json
+++ b/packages/next-adapter/package.json
@@ -11,9 +11,6 @@
     "clean": "rimraf build ./tsconfig.tsbuildinfo",
     "lint": "eslint ."
   },
-  "bin": {
-    "next-expo": "echo \"the next-expo command has been deprecated in favor of manual setup. See README for more.\" && exit 1"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/expo/expo-cli.git",

